tests/FileTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * This file is part of the PHP Rebilly API package.
+ *
+ * (c) 2015 Rebilly SRL
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Rebilly\Tests;
+
+use Rebilly\ApiKeyProvider;
+use Rebilly\Client;
+use Rebilly\Rest\File;
+use RuntimeException;
+
+/**
+ * Class FileTest.
+ *
+ * @group wip
+ */
+class FileTest extends TestCase
+{
+    public function testFile()
+    {
+        $client = new Client(['apiKey' => 'QWERTY']);
+
+        $client = new Client(
+            [
+                'apiKey' => ApiKeyProvider::env(),
+                'httpHandler' => function () use ($client) {
+                    $response = $client
+                        ->createResponse()
+                        ->withHeader('Content-Type', 'application/pdf')
+                    ;
```

Comment:
it's not the same... how many time will there be a need to add a new array element in contrast to how many time will you add a new method call to a chain?

Btw, there are also code that puts a semicolon on a separate line after a ternary operator like:
```
$user = $id > 0
    ? $user->findById($id) //imagine this longer
    : null
;
```
I really don't see any reason to have a semicolon alone in one row.